client/components/account-balances/index.tsx

```diff
@@ -84,16 +84,16 @@ const AccountBalances: React.FC = () => {
 						className="wcpay-account-balances__balances"
 					>
 						<BalanceBlock
-							id={ `wcpay-account-balances-${ tab.currencyCode }-available` }
-							title={ fundLabelStrings.available }
-							amount={ tab.availableFunds }
+							id={ `wcpay-account-balances-${ tab.currencyCode }-total` }
+							title={ fundLabelStrings.total }
+							amount={ 0 }
 							currencyCode={ tab.currencyCode }
 							isLoading
 						/>
 						<BalanceBlock
-							id={ `wcpay-account-balances-${ tab.currencyCode }-pending` }
-							title={ fundLabelStrings.pending }
-							amount={ tab.pendingFunds }
+							id={ `wcpay-account-balances-${ tab.currencyCode }-available` }
+							title={ fundLabelStrings.available }
+							amount={ 0 }
```

Comment:
In loading state, available balance is 0.

Reply:
In loading state merchant will see a grey box placeholder, so this `0` is all good (if it shows in UI could lead to questions).

Reply:
<img width="740" alt="Screenshot 2024-02-19 at 1 34 05 PM" src="https://github.com/Automattic/woocommerce-payments/assets/4167300/70734b42-2254-4e95-8732-84adb694867a">